frontend/src/components/login.js

```diff
@@ -1,44 +1,165 @@
 import { useState } from 'react';
-import { loginUser } from '../http/user'
+import { registerUser, loginUser } from '../http/user'
 
-const Login = ({ setUserState }) => {
+const Login = ({ setUserState, setMessage, setReturnError }) => {
+    //login states
```

Comment:
//login states => // login form states. 

To the average reader **login states** will probably be interpreted as is the user logged in or not? The comment is ambiguous.

Reply:
changed